### ISSUE TITLE
Crash after tabbing back to game and moving

### ISSUE THREAD

**Issue by fitqt** [Soreepeong/XivAlexander#310]
[https://i.imgur.com/xUIR8my.png](url)

**Comment by Soreepeong** [comment]
Images do not help. Export log does.

**Comment by fitqt** [comment]
where's the log location?

**Comment by Soreepeong** [comment]
There's an Export Log button in that window.

**Comment by fitqt** [comment]
well I closed the window...do I need to try triggering it again if I dont export it at that time?

**Comment by Soreepeong** [comment]
Yeah, you will have to do that. XivAlexander does not save logs by default.

**Comment by ShinjukuCurry** [comment]
Same issue, crashed after alt-tab. Fortunately I did export the log file here.
[XivAlexander_20220110_224337_520.log](https://github.com/Soreepeong/XivAlexander/files/7843558/XivAlexander_20220110_224337_520.log)

**Comment by Soreepeong** [comment]
OP: Are you sure that you pressed Alt+Tab? If then, are you using some sort of hotkey/window manager? Alt+Tab by itself cannot trigger XivAlexander from unloading. I'd guess that this is some other program being at fault.

ShinjukuCurry: DirectX errors are never about XivAlexander. Make sure that integrity check using FFXIVQuickLauncher passes, and if then, you'll have to try disabling Dalamud and see if it crashes, and vice versa with XivAlexander.

**Comment by fitqt** [comment]
> OP: Are you sure that you pressed Alt+Tab? If then, are you using some sort of hotkey/window manager? Alt+Tab by itself cannot trigger XivAlexander from unloading. I'd guess that this is some other program being at fault.

I'm pretty sure I just did quick Alt+Tab in that case, but I also use this plugin: [https://github.com/Eternita-S/Mini](url) which allows minimizing the game "properly" so if you have time to check could it interfere with alex would be nice.
